### ISSUE TITLE
Fence jumping doesn't seem to work

### ISSUE THREAD

**Issue by foopex** [AllTheMods/ATM-3#340]

**Comment by OrdinatorStouff** [comment]
Will need to test/confirm this and also see if the mod has an issue tracker with this reported if confirmed.

**Comment by FallenEagle1** [comment]
Did a little testing, maybe this covers the issues found. From the testing, it looks like jumping over fences is working in the environments I tested. The two environments I used were:

ATM3 v.5.6 (fence jumper 1.0.1)
Customer pack with only forge 14.23.1.2583 and fence jumper 1.0.1

The testing done was with a couple fences from each mod found in ATM3. The one issue I found in that testing was while fences worked, gates would not work under specific situations. If a gate was within one block of a fence in any x or z direction(corner way works also), it could be jumped over. If it was not within these parameters, the gate could not be jumped over, even if it touched a gate that could be jumped over.
